app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -214,4 +222,34 @@ class DradisDatatable {
       that.dataTable.destroy();
     });
   }
+
+  selectedIds() {
+    var selectedRows = this.dataTable.rows({ selected: true });
+    var ids = selectedRows.ids().toArray().map(function(id) {
+      // The dom id for <tr> is in the following format: <tr id="item_name-id"></tr>,
+      // so we split it by the delimiter to get the id number.
+      var idArray = id.split('-');
+      return idArray[1];
+    });
+
+    return ids;
+  }
+
+
+  // -- Merge --------------------------------------------------------
+
+  toggleMergeBtn(isShown) {
+    var mergeBtn = this.dataTable.buttons('mergeBtn:name');
```

Comment:
Can we use a guard instead? Similar to https://github.com/dradis/dradis-ce/pull/847/files#diff-eec3b3902c4b9d413904c8dbc2f8f3aa7814a2b9ca52316a6315a643ee7ffc7aR135-R137